### PR TITLE
Add two classes from GraphStream to precision/recall test suite, and fix goal file for Player (FreeCol)

### PR DESCRIPTION
**Pull request by ariannab**

### CODE REVIEW

**Comment by ariannab** [comment]
Closed this temporarily due to problems with Travis checks. 
I will reopen it at a later time.

File: src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json [hunk 1]
```diff
@@ -8437,7 +8437,7 @@
     "parameters": [],
     "returnTag": {
       "comment": "True if this object has been disposed.",
-      "condition": "",
+      "condition": "true ? result=target.disposed",
```
Comment (albertogoffi) [inline src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json]: If I'm not wrong, disposed is a private field in a superclass, then is not visible.

File: src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json [hunk 2]
```diff
@@ -8413,7 +8413,7 @@
     "parameters": [],
     "returnTag": {
       "comment": "True if this object is not initialized.",
-      "condition": "(target.isUninitialized()) == false ? result==true",
+      "condition": "true ? result==target.uninitialized",
```
Comment (albertogoffi) [inline src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json]: If I'm not wrong, disposed is a private field in a superclass, then is not visible.

File: src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json [hunk 3]
```diff
@@ -2128,7 +2128,7 @@
     ],
     "returnTag": {
       "comment": "True if the player can afford the purchase.",
-      "condition": "",
+      "condition": "target.gold >= args[0] ? result==true",
```
Comment (albertogoffi) [inline src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json]: I think this translation comes from the code, but is actually difficult (impossible?) to infer from the comment itself.
If this is the case, I suggest to leave the translation empty.

**Pull request merged**